setup.py

```diff
@@ -15,7 +15,7 @@
     entry_points={'console_scripts': ['wooify = wooey.backend.command_line:bootstrap', ]},
     python_requires='>3.5.0',
     install_requires=[
-        'celery>=4.0,<6',
+        'celery>=4.0,<5',
```

Comment:
Can you elaborate on the `-A` option? Might need to update docs and the bootstrap for this

Reply:
Or separate this into another MR in case we do need that.

Reply:
It looks like it's still valid, how are we calling it at ginkgo? https://docs.celeryq.dev/en/stable/django/first-steps-with-django.html#starting-the-worker-process

Reply:
It looks like the`-A` option moved to become a global option, starting 5.0
```
wooey-celery-1  | Error: No such option: -A
wooey-celery-1  | You are using `-A` as an option of the worker sub-command:
wooey-celery-1  | celery worker -A celeryapp <...>
wooey-celery-1  |
wooey-celery-1  | The support for this usage was removed in Celery 5.0. Instead you should use `-A` as a global option:
wooey-celery-1  | celery -A celeryapp worker <...>
```

Reply:
Is there a reason why we can't pin `celery>=5.0` and move `-A` to be global?